conf/ConfigOptions.asciidoc

```diff
@@ -423,21 +423,21 @@ The spacing used by the EdgeDistanceExtractor.  Units in meters.
 
 === element.cache.size.node
 
-* Data Type: long
+* Data Type: int
```

Comment:
@bwitham Can you see my comment on this change and tell if it'll cause problems on the command line?  (i.e. How is the Data Type set in this file used in the core code?)